npm/builder.js

```diff
@@ -27,10 +27,16 @@ function Builder () {
     'node_modules/opal-runtime/src/opal.js',
     'build/asciidoctor-lib.js'
   ];
-  if (process.env.ASCIIDOCTOR_CORE_VERSION) {
-    this.asciidoctorCoreVersion = process.env.ASCIIDOCTOR_CORE_VERSION;
+  var asciidoctorRef = process.env.ASCIIDOCTOR_CORE_VERSION;
+  if (asciidoctorRef && ~asciidoctorRef.indexOf('/')) {
+    var segments = asciidoctorRef.split('/');
+    this.asciidoctorCoreUser = segments[0];
+    segments = segments[1].split('#');
+    this.asciidoctorCoreRepo = segments[0];
```

Comment:
`asciidoctorCoreRepo` should always be `asciidoctor` right ?

Reply:
If we're going to allow the form user/repo#branch, we should do it properly and not make any assumptions.